src/System Application/Test/Rest Client/app.json

```
@@ -35,7 +35,7 @@
   "idRanges": [
     {
       "from": 134970,
-      "to": 134974
+      "to": 134975
```

This ID is already used by codeunit 134975 "ERM Dimension Report". 134977 seems to be available.
```suggestion
      "to": 134977
```